mtp_api/apps/core/templatetags/tick_charts.py

```diff
@@ -33,7 +33,7 @@ def get_zero(value):
         return datetime.timedelta()
     if isinstance(value, (int, float)):
         return 0.0
-    raise ValueError('Cannot determine zero value for a %s' % type(value))
+    raise ValueError('Cannot determine zero value', {'type': type(value)})
```

Comment:
I wonder if add the value to the Sentry context would also be useful to debug/investigate.
I tend to prefer having a bit more information, as it helps with debugging problems but obviously this is not wrong or anything ☺️